services/bots/src/discord/listeners/line_count_enforcer.ts

```diff
@@ -0,0 +1,46 @@
+import { On } from '@discord-nestjs/core';
+import { Message, AttachmentBuilder } from 'discord.js';
+
+export const MAX_LINE_LENGTH = 17;
+const KNOWN_FILETYPES = new Set([
+  'javascript',
+  'js',
+  'json',
+  'ts',
+  'txt',
+  'typescript',
+  'yaml',
+  'yml',
```

Comment:
I think adding these might be useful too

```suggestion
  'diff',
  'javascript',
  'js',
  'json',
  'py',
  'sh',
  'ts',
  'txt',
  'typescript',
  'yaml',
  'yml',
```